ml_enabler/tests/fixtures/geojson.py

```diff
@@ -0,0 +1,105 @@
+def get_geojson():
```

Comment:
see nitpicky comment below about truncating decimals in lat/lng values in tests. not a blocker to merging, of course.